apps/webcert/src/page/CertificatePage.tsx

```diff
@@ -82,6 +83,7 @@ const CertificatePage: React.FC = () => {
         !isCertificateDeleted && (
           <>
             <MajorVersionNotification />
+            <InactiveCertificateTypeNotification />
```

Comment:
The comment is not on the correct line, but I add it here as MajorVersionNotification is added here.

Could we add logic to MajorVersionNotification that it is only displayed if inactiveCertificateType not true?